jenkinsfiles/install-android-sdk.sh

```diff
@@ -0,0 +1,24 @@
+set -x
+
+BASE_DIR=/opt/tools
+ANDROID_SDK_DIR=/android-sdk-linux
+
+if [ -d "${BASE_DIR}/${ANDROID_SDK_DIR}" ]
+then
+  echo 'Android SDK installed continueing normally'
+else
+  echo 'Installing Android SDK'
+  cd ${BASE_DIR}
+  wget --output-document=android-sdk.tgz --quiet https://dl.google.com/android/android-sdk_r24.4.1-linux.tgz
+  tar xzf android-sdk.tgz 
+  rm -f android-sdk.tg
+  
+  ( sleep 5 && while [ 1 ]; do sleep 1; echo y; done ) | android-sdk-linux/tools/android update sdk --all --no-ui --filter platform-tools,tools
```

Comment:
[Question] Why we need this `( sleep 5 && while [ 1 ]; do sleep 1; echo y; done )`